xml/System.Buffers/SequenceReader`1.xml

```diff
@@ -29,8 +29,8 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
+    <typeparam name="T">The type of the read-only sequence.</typeparam>
```

Comment:
The type definition is incorrect since it says `T : struct`. It should be `unmanaged`
![image](https://user-images.githubusercontent.com/6527137/62092630-b796e180-b22a-11e9-9376-93eab111d639.png)

Is this something that can be fixed?
https://github.com/dotnet/corefx/blob/fec76b0fc06b4fce9e4ac28b9a2bf1101209d984/src/System.Memory/src/System/Buffers/SequenceReader.cs#L12

Also, what about the other generic constraint that `T  implements IEquatable<T>`

Reply:
@mairaw @rpetrusha @joelmartinez this is more of a question for you. Ahson found that Docs is not showing what we have in docs.

@ahsonkhan would you still agree with the documentation I added? If that's the case, we can open a separate issue to address the definition problem.

Reply:
@dend and @joelmartinez, it seems that the generic type constraints are incorrect here. Instead of `where T : Struct`, it should be `where T : Unmanaged, IEquatable<T>`

Reply:
> would you still agree with the documentation I added?

Other than the feedback, yes, the docs look good. 

> If that's the case, we can open a separate issue to address the definition problem.

Sure.

Reply:
Okay thanks. I'll mark this as resolved since it's an external issue, unrelated to this PR.

Reply:
Is there an issue tracking this?

Reply:
/fyi @mimisasouvanh This looks like an enhancement will need to be made to the C# type signature formatter in mdoc to add support for unmanaged type constraints.